repository/f.json

```diff
@@ -521,6 +521,18 @@
 				}
 			]
 		},
+		{
+			"name": "File Filter",
+			"details": "https://github.com/vsc-cnst/SublimeTextFileFilter",
+			"author": "vsc-cnst",
```

Comment:
```suggestion
```

Just one final thing: if we're going with your GitHub username here, we don't even need it here, it's automatic.